packages/stuffex-frontend/src/MyApp.js

```diff
@@ -4,18 +4,20 @@ import Item from './Item'
 import React, { useState, useEffect } from 'react'
 import { Route, Routes } from 'react-router-dom'
 import Form from './components/Form'
+import Profile from './components/Profile'
 import Navbar from './components/Navbar'
 import About from './components/About'
 import ItemPage from './ItemPage'
 import UserItems from './UserItems'
 
 function MyApp() {
   const [items, setItems] = useState([])
+  const [users, setUsers] = useState([])
 
   const backendRoot = 'https://stuffex.azurewebsites.net'
 
   function populateItems(query) {
-    fetchUsers(query)
+    fetchItems(query)
```

Comment:
Good catch, I don't know why that was named `fetchUsers`, and it would have gotten really confusing if you hadn't fixed that name.